src/homepageExperience/containers/CliWizard.tsx

```diff
@@ -63,13 +63,6 @@ export class CliWizard extends PureComponent<{}, State> {
   }
 
   handleNextClick = () => {
-    this.setState({
-      currentStep: Math.min(
-        this.state.currentStep + 1,
-        HOMEPAGE_NAVIGATION_STEPS_SHORT.length
-      ),
-    })
-
     event(
       'firstMile.cliWizard.next.clicked',
       {},
```

Comment:
seems like we're missing the metadata here? (the data about what step the user is on, and where was the next clicked?

Reply:
This was github minimizing the next code lines, sorry for the false alarm.